### PR TITLE
Allow a new user to not encounter error on their first deploy

### PR DESCRIPTION
**Pull request by Bachibouzouk**
- Create a separate `requirements.txt` file and include it in `environment.yml`.
- Improve the documentation steps and clarity for a fresh first install.
- Make sure one can at least execute `python manage.py runserver` and see the WAM welcome page, even if no apps are provided (i.e. if `WAM_APPS` environment variable is set to an empty string).
- Provide a template for the config file location.
- Change environment variable name `CONFIG_PATH` to `WAM_CONFIG_PATH`.

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -1,16 +1,5 @@
-name: django
+name: wam-django
```
Comment (Bachibouzouk) [inline environment.yml]: I found `django` to be too broad...

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+django
+jmespath
```
Comment (Bachibouzouk) [inline requirements.txt:2]: This dependency was missing

File: wam/settings.py [hunk 3]
```diff
@@ -15,7 +15,14 @@
 import logging
 from configobj import ConfigObj
 
-config = ConfigObj(os.environ['CONFIG_PATH'])
+config_file_path = os.environ.get('WAM_CONFIG_PATH')
```
Comment (Bachibouzouk) [inline wam/settings.py:18]: The builtin `dict` method `get` will return `None` if the key does not exist.

**Comment by Bachibouzouk** [comment]
@rl-institut/wam-team  - can you let me know what you think of these changes?

File: environment.yml (hunk 1 above)
Comment (christian-rli) [reply environment.yml]: I agree, but will this have an effect on already installed instances of WAM (i.e. our server?)

Comment (Bachibouzouk) [reply environment.yml]: The environment name `django` is also used in the `docker-compose.yml` file. Although we could also change `django` to `wam-django`, I decided to revert this change for the time being (a41ce1ced6292227342befe52ef9720f2ff52670)

**Comment by Bachibouzouk** [comment]
@christian-rli - can you review this PR ?

**Review by christian-rli — APPROVED** [review]
We can change this to wam-django when @henhuy is back

**Pull request merged**